eng/common/templates/jobs/vmr-synchronization.yml

```diff
@@ -0,0 +1,76 @@
+### These steps synchronize code from product repositories into the VMR (https://github.com/dotnet/dotnet)
```

Comment:
Is this file going to get backported to Arcade?  If so what is the value in porting it to arcade?

Reply:
Ah, I moved the two YAMLs from `src/VirtualMonoRepo` into `/eng` and the job template ended up in `common`.

I'd say there's no reason porting this to Arcade, not for now at least. What would be a good place for it here?

`/eng/vmr-synchronization-job.yml`
?

Reply:
Agreed, I don't see a reason to move into Arcade.  I personally prefer adding a eng/pipelines/templates/... folder structure in installers.